astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+#### App Settings
+Update the `appsettings.json` file. You need to add the entire `complete-application` object so that the code above can be configured correctly. `Authority` is just the location of the user identity server, in this case, FusionAuth.  `Urls` ensures the application runs on the specified port, 5000.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/appsettings.json"}
+  lang="json" />
+
+#### Program Settings
+Update the `Program.cs` file. Since you are using unsecured cookies, you need to allow this in the configuration.  Safari and Chrome require different settings and that is reflected in the code below.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Program.cs"}
+  lang="c#" />
+
+#### Startup Settings
+Add the `Startup.cs` file in the root directory of the application. This sets the correct port and manages unsecured cookies for the browsers.  The `CheckSameSite` and `DisallowsSameSiteNone` functions help manage the cookie settings for the different browsers.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Startup.cs"}
+  lang="c#" />
+
+<Aside type="note">
+There are a few settings applied to simplify the sample application. In a production application, do not allow cookies over `http` or keep the `ClientSecret` in a .json file for production applications.  Get the `ClientSecret` from an environmental variable that has been set on a production machine or some other secure way.
+</Aside>
+
+#### Navigation
+This application will display different menu items for authenticated users and non authenticated users. To add this type of navigation update `/Pages/Shared/_Layout.cshtml` with the following:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Shared/_Layout.cshtml"}
+  lang="razor" />
+
+#### Homepage
+Update the default page.
+
+Copy the following code into `Index.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `Index.cshtml` and notice the logic to show a different view to users that have not logged in.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml"}
+  lang="razor" />
+
+#### Styling
+To give the pages the look and feel of an application, apply the following styling and images.
+
+Add the `changebank.css` file in the `wwwroot\css" directory of the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/wwwroot/css/changebank.css"}
+  lang="css" />
+
+Add the Changebank logo to the application.
+
+Copy the money image for the `Index` page.
+
+```shell
+curl --create-dirs --output wwwroot/img/money.jpg  https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-web/main/complete-app/app/assets/images/money.jpg
+```
+
+## Conclusion
+
```

Comment:
I tried to do this and got these error messages (I was running in a new directory, `new-application` but did change the namespace references):

```
dan@mooreds new-application % dotnet publish -r osx-x64
MSBuild version 17.7.1+971bf70db for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
/usr/local/share/dotnet/sdk/7.0.400/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(1156,5): warning NETSDK1179: One of '--self-contained' or '--no-self-contained' options are required when '--runtime' is used. [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Pages/Account.cshtml.cs(10,30): error CS0246: The type or namespace name 'PrivacyModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Pages/Account.cshtml.cs(12,34): error CS0246: The type or namespace name 'PrivacyModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Account_cshtml.g.cs(60,71): error CS0246: The type or namespace name 'AccountsModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Logout_cshtml.g.cs(51,71): error CS0246: The type or namespace name 'LogoutModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Index_cshtml.g.cs(60,71): error CS0246: The type or namespace name 'IndexModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Account_cshtml.g.cs(62,81): error CS0246: The type or namespace name 'AccountsModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Logout_cshtml.g.cs(53,81): error CS0246: The type or namespace name 'LogoutModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Index_cshtml.g.cs(62,81): error CS0246: The type or namespace name 'IndexModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Account_cshtml.g.cs(63,16): error CS0246: The type or namespace name 'AccountsModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Logout_cshtml.g.cs(54,16): error CS0246: The type or namespace name 'LogoutModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]
/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/Microsoft.NET.Sdk.Razor.SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator/Pages_Index_cshtml.g.cs(63,16): error CS0246: The type or namespace name 'IndexModel' could not be found (are you missing a using directive or an assembly reference?) [/Users/dan/dev/inversoft/fusionauth/fusionauth-quickstart-dotnet-web/new-application/new-application.csproj]

```

Reply:
Ah, this was due to my not fully doing the renaming of the package.

Reply:
trying to remove this kind of confusion from the project. It is not necessarily normal, but it will make the sample better.

Reply:
changed the application behavior to hopefully remove this confusion.